llvm/lib/Target/RISCV/RISCVISelLowering.cpp

```diff
@@ -16079,7 +16079,7 @@ static SDValue expandMul(SDNode *N, SelectionDAG &DAG,
   uint64_t MulAmt = CNode->getZExtValue();
```

Comment:
I'm a little concerned that some of this function treats the constant as unsigned, and some as signed, but I don't know if that instinct is right.

Reply:
My instincts are wrong, nvm.